app-antivirus/clamav/clamav-1.2.0_rc.ebuild

```diff
@@ -0,0 +1,398 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LLVM_MAX_SLOT=13
```

Comment:
Uuh...

Reply:
There's a reason for the LLVM max slot - clamav has a LLVM-based bytecode interpreter which (unfortunately) only supports LLVM that's no longer in tree.

I would remove the capability from the ebuild but upstream are working on it though we're only likely to see that for 1.3.0

Reply:
IF the upstream is working on, I'd say it can stay. Please put a comment above it linking the upstream issue. Let's not remove code if it becomes valid again, but if it's utterly broken and no hope of ever getting fixed, we should just clean the llvm parts from this ebuild and unconditionally `--disable-jit`.

```suggestion
# 13 is correct, upstream is working on it.
# www-link-to-upstream-issue
# 2023-08-12
LLVM_MAX_SLOT=13
```